.github/workflows/main.yml

```diff
@@ -66,18 +66,69 @@ jobs:
           ninja
           ninja test
 
-      - name: powerloader python based tests
-        shell: bash -l -eo pipefail {0}
-        if: runner.os != 'Windows'
+      - name: Setup minio
         env:
-          AWS_ACCESS_KEY: ${{ secrets.AWS_ACCESS_KEY }}
-          AWS_SECRET_KEY: ${{ secrets.AWS_SECRET_KEY }}
-          AWS_DEFAULT_REGION: ${{ secrets.AWS_DEFAULT_REGION }}
+          AWS_S3_BUCKET: ${{testbucket}}
+          # AWS_ACCESS_KEY_ID: ${{minioadmin}}
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
```

Comment:
if we use MINIO please hardcode the KEY_ID to `minioadmin` and don't use the real keys.